week3/solutions/odd_tuples.py

```diff
@@ -0,0 +1,18 @@
+'''
+Write a procedure called oddTuples, which takes a tuple as input,
+and returns a new tuple as output, where every other element of 
+the input tuple is copied, starting with the first one. So if test 
+is the tuple ('I', 'am', 'a', 'test', 'tuple'), then evaluating 
+oddTuples on this input would return the tuple ('I', 'a', 'tuple')
+'''
+
+def odd_tuples(tuple):
```

Comment:
`tuple` is a type in Python. It's not wise to override that type with your name, here.